### PR TITLE
feat: add banner support

### PR DESCRIPTION
**Pull request by Soare-Robert-Daniel**
### Summary

Add SDK banner support.

### Will affect visual aspect of the product

YES

### Screenshots 
image
### Test instructions

Must be used with https://github.com/Codeinwp/themeisle-sdk/pull/269

- Set the instance date between 25 NOV and 3 DEC to trigger Black Friday banner
- You should see the banner on the Library page.
- You should NOT see the banner if it is not a Blackfriday Interval

## Check before Pull Request is ready:

* [ ] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

Closes https://github.com/Codeinwp/themeisle/issues/1688

### CODE REVIEW

File: classes/Visualizer/Module/Admin.php [hunk 1]
```diff
@@ -1093,6 +1093,10 @@ public function renderLibraryPage() {
 
 		$this->load_survey();
 
+		if ( ! apply_filters( 'visualizer_is_business', false ) ) {
```
Comment (abaicus) [inline classes/Visualizer/Module/Admin.php:1096]: Are we 100% sure that this is the only condition that marks the product as pro? To me it sounds that this only applies to the business plan. 

I am not that familiar with this plugin's codebase so maybe it's worth checking other places where free-only things are rendered.

For reference, this is the check we do when adding the pro upsell: 

https://github.com/Codeinwp/visualizer/blob/b1c71ac778ecacfd74263b82e4c6e006adb8cd01/classes/Visualizer/Module/Sources.php#L138-L139

**Review by abaicus — COMMENTED** [review]
@Soare-Robert-Daniel I just left one thing. If you're sure the condition looks good and applies to all pro users, then it's fine. I'd still review it though.

File: classes/Visualizer/Module/Admin.php (hunk 1 above)
Comment (Soare-Robert-Daniel) [reply classes/Visualizer/Module/Admin.php:1096]: The code for that is here https://github.com/Codeinwp/visualizer-pro/blob/77e1836ad404589168902b90e4cfaba36e5927f6/inc/addon.php#L2255-L2274

If the filters return false, that the user is free and does not have the Pro module, or if it has the pro module, false if it  is below 1 or equal to 10.

If it returns true, then we have a pro user that is above Tier 1 Pro

**Review by abaicus — APPROVED** [review]

File: classes/Visualizer/Module/Admin.php (hunk 1 above)
Comment (abaicus) [reply classes/Visualizer/Module/Admin.php:1096]: Thanks for the explanation! 👍🏻

**Pull request merged**